commit ec031020eb7ed9a0cc3d086bcd7ae7611eb308fc
Author: Panu Matilainen <pmatilai@redhat.com>
Date:   Mon Feb 7 09:15:29 2011 +0200

    Fix segfault when building more than one rpm (RhBug:675565)
    - Remember to set the static string cache pointer to NULL on free, duh

diff --git a/lib/rpmug.c b/lib/rpmug.c
index 2bb18e5..b365e4a 100644
--- a/lib/rpmug.c
+++ b/lib/rpmug.c
@@ -198,5 +198,5 @@ void rpmugFree(void)
     rpmugGid(NULL, NULL);
     rpmugUname(-1);
     rpmugGname(-1);
-    strCacheFree(strStash);
+    strStash = strCacheFree(strStash);
 }
